### PR TITLE
🌱 Release v1.6.0-beta.1

### PR DESCRIPTION
**Pull request by g-gaston**
/area release

Will remove the redundant dependency entries once we have the first round of reviews, in case I need to regenerate the notes

### CODE REVIEW

File: CHANGELOG/v1.6.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+- ClusterClass: Improve Cluster variable defaulting/validation errors (#9452)
+- ClusterClass: Improve message for TopologyReconciledCondition (#9400)
+- ClusterClass: Make ClusterClass generated object names consistent (#9254)
+- ClusterClass: Minor fixes for CC+MP implementation (#9318)
+- clusterctl: Check resource blocking clusterctl move during discovery (#9246)
+- clusterctl: Use goproxy to check version in clusterctl (#9237)
+- clusterctl: Use http get to download files from GitHub in clusterctl (#9236)
+- Dependency: Bump cert-manager to v1.12.4 (#9358)
+- Dependency: Bump cert-manager to v1.13.0 (#9408)
+- Dependency: Bump cert-manager to v1.13.1 (#9505)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.6.0-beta.1.md:186]: same

File: CHANGELOG/v1.6.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+- Dependency: Bump cloud.google.com/go/storage from 1.29.0 to 1.31.0 in /hack/tools (#9053)
+- Dependency: Bump cloud.google.com/go/storage from 1.31.0 to 1.32.0 in /hack/tools (#9218)
+- Dependency: Bump cloud.google.com/go/storage from 1.32.0 to 1.33.0 in /hack/tools (#9423)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.6.0-beta.1.md:189]: merge into 1

File: CHANGELOG/v1.6.0-beta.1.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+- Dependency: Bump controller tools to v1.13.0 (#9221)
+- Dependency: Bump controller-runtime to v0.15.1 (#9126)
+- Dependency: Bump controller-runtime to v0.16.3 (#9592)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.6.0-beta.1.md:192]: merge into 1

**Review by furkatgofurov7 — COMMENTED** [review]
Thanks, few nits + we need to merge dependency bumps entries into 1 (did not want to comment in all occurrences as I was going down the line and stopped it 😄)

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Review by furkatgofurov7 — COMMENTED** [review]
/hold

**Review by nawazkh — APPROVED** [review]
/lgtm

**Review by furkatgofurov7 — COMMENTED** [review]
/hold cancel

**Pull request merged**